src/sentry/notifications/defaults.py

```diff
@@ -34,7 +34,22 @@
     NotificationSettingTypes.QUOTA_WARNINGS: NotificationSettingOptionValues.ALWAYS,
 }
 
+NOTIFICATION_SETTINGS_MSTEAMS = {
+    NotificationSettingTypes.DEPLOY: NotificationSettingOptionValues.NEVER,
+    NotificationSettingTypes.ISSUE_ALERTS: NotificationSettingOptionValues.NEVER,
+    NotificationSettingTypes.ACTIVE_RELEASE: NotificationSettingOptionValues.NEVER,
+    NotificationSettingTypes.WORKFLOW: NotificationSettingOptionValues.NEVER,
+    NotificationSettingTypes.APPROVAL: NotificationSettingOptionValues.NEVER,
```

Comment:
@97varun Certain settings should just be enabled by default such as `NotificationSettingTypes.APPROVAL`, you can follow what we do for Slack.

Reply:
@scefali These notifications aren't implemented and enabling these types before deploying the UI changes makes the settings UI behave weirdly, so I have kept them disabled till we introduce the UI changes and support these notification types.